runtime/caml/stack.h

```diff
@@ -94,6 +94,16 @@
 #define Stack_header_size 32
 #endif
 
+#ifdef TARGET_loongarch64
```

Comment:
This file needs changes as done in #12275.

I am surprised that you have been able to compile this after your rebase to `47a0a17a516b58d8ac1444f25b25f653a87be263`, as the code requires at least the definition of the `First_frame` macro in this section. Or did you compile with `TARGET_riscv` defined by mistake?

Reply:
sorry. I'm missing a patch.